File: Exercício_28.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    int i, num;
    i = 0;

    printf("Digite um numero: ");
    scanf("%d", &num);

    for (int i = 0; i <= 10; i++)
    {
    printf("%d x %d = %d\n", num, i, i * num);
    }

    system("pause");

    for (int i = 10; i <= 20; i++)
    {
    printf("%d x %d = %d\n", num, i, i * num);
    }

 return 0;
}

File: Exercício_28.py
num = int(input('digite um numero para a tabuada '))
num = 1
i = 1
while(i<=19):
    i = i + 1
    for num in range(1, 11, 1):
        r = num * i
        print(f'{i} X {num} = {r}')
        num = num + 1
        if(num>10):
            print("\n")